src/resources/BreadcrumbResources.ts

```diff
@@ -30,33 +29,26 @@ export function getMyBusinessRegistryBreadcrumb (businessUrl: string): Breadcrum
   }
 }
 
-/** Returns the breadcrumb to the Staff dashboard. */
+/** Returns the breadcrumb to the Staff Dashboard. */
 export function getStaffDashboardBreadcrumb (businessUrl: string): BreadcrumbIF {
   return {
     text: 'Staff Dashboard',
     href: `${businessUrl}staff/dashboard/active`
   }
 }
 
-/** Returns the breadcrumb to the Digital Credentials page. **/
+/** Returns the breadcrumb to the Business Digital Credentials. */
 export function getDigitalCredentialBreadcrumb (): BreadcrumbIF {
   return {
     text: 'Business Digital Credentials',
     to: { name: Routes.DIGITAL_CREDENTIALS }
   }
 }
 
-/** Returns the breadcrumb to the appropriate dashboard based on the feature flag. */
+/** Returns the breadcrumb to the Business Dashboard. */
 export function getDashboardBreadcrumb (entityName: string, businessDashUrl: string, identifier: string): BreadcrumbIF {
-  if (GetFeatureFlag('use-business-dashboard')) {
-    return {
-      text: entityName || 'Unknown Name',
-      href: `${businessDashUrl}/${identifier}`
-    }
-  } else {
-    return {
-      text: entityName || 'Unknown Name',
-      to: { name: Routes.DASHBOARD }
```

Comment:
I need to fix this. The old code route to a local route (which is fast and efficient) while the new code redirects to an URL, which requires a full app reload.